media/css/newsletter/newsletter-developer.scss

```diff
@@ -0,0 +1,231 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../pebbles/includes/lib';
+
+.content {
+    @include clearfix;
+
+    @media #{$mq-tablet} {
+        width: $width-tablet;
+    }
+
+    @media #{$mq-desktop} {
+        width: $width-desktop;
+    }
+}
+
+.section-intro {
+    @include background-size(60px 60px, 200px 199px);
+    background-color: #0c99d5;
+    background-image: url('/media/img/newsletter/developer/grid.svg'),
+                      url('/media/img/newsletter/developer/heart.svg');
+    background-position: top left, top 30px center;
+    background-repeat: repeat, no-repeat;
+    height: 230px;
+    width: 100%;
+
+    @media #{$mq-phone-wide} {
+        @include background-size(80px 80px, 260px 258px);
+        height: 290px;
+    }
+
+    @media #{$mq-tablet} {
+        @include background-size(90px 90px, 320px 318px);
+        background-position: top left, top 20px center;
+        height: 340px;
+    }
+
+    @media #{$mq-tablet} and #{$mq-short} {
+        @include background-size(80px 80px, 260px 258px);
+        height: 290px;
+    }
+
+    .content {
+        position: relative;
```

Comment:
`.content` is already relative in the pebbles base so I don't think you need to redeclare it here.